tests/src/rules/no-duplicates.js

```diff
@@ -129,6 +129,30 @@ ruleTester.run('no-duplicates', rule, {
       errors: ['\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.'],
     }),
 
+    // #2347: duplicate identifiers should be removed
+    test({
+      code: "import {a} from './foo'; import { a } from './foo'",
+      output: "import {a} from './foo'; ",
+      errors: ['\'./foo\' imported multiple times.', '\'./foo\' imported multiple times.'],
+      parser: parsers.BABEL_OLD, // Needed to avoid "A fatal parsing error occurred: Parsing error: Identifier 'a' has already been declared"
```

Comment:
for all of these "babel old" test cases, do they *also* work on babel new, or on either of the two TS parsers?

if so, let's duplicate the test cases and cover each parser it works on :-)

Reply:
Done in d9ca6a238c283e23a614fce33a2e7aa327ff0749!